src/Adapter/Tools.php

```diff
@@ -126,16 +126,4 @@ public function getShopDomainSsl($http = false, $entities = false)
     {
         return LegacyTools::getShopDomainSsl($http, $entities);
     }
-
-    /**
-     * Get file contents from URL
-     *
-     * @param string $url
-     *
-     * @return bool|string
-     */
-    public function getFileContents($url)
```

Comment:
we can't remove it, it's part of public API. But you can deprecate it.

Reply:
it's not part of public API, I have added it and now removed as it's not needed anymore.